packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -216,13 +217,17 @@ QUnit.module('collapsible groups', moduleSetup, () => {
 
         const instance = $element.dxList('instance');
 
-        $element.trigger('focusin');
 
         $headers.eq(1).trigger('dxclick');
         $headers.eq(1).trigger('dxclick');
+        $element.trigger('focusin');
```

Comment:
Why did u move it? I prefer we do not change logic of old tests, it's just more safe